packages/guideline-blocks-settings/src/hooks/useAttachments.tsx

```diff
@@ -1,6 +1,9 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { AppBridgeBlock, Asset, useBlockAssets } from '@frontify/app-bridge';
+import { type ReactNode, createContext, useContext } from 'react';
+
+import { type BlockProps } from 'src';
```

Comment:
```suggestion
import { type BlockProps } from '../';
```

Had some bad experience with src imports 😃

Reply:
I actually couldn't import this from anywhere else because it's in the src index file, but I will check again

Reply:
fixed